adaptdl/adaptdl/torch/parallel.py

```diff
@@ -64,14 +59,29 @@ def __init__(self, model, optimizer, lr_scheduler=None,
             param.register_hook(functools.partial(self._backward_hook, param))
 
         # Setup for AdaScale, must be after registering backward hooks!
-        self.adascale = AdaScale(optimizer, patch_optimizer=True)
+        self.adascale = AdaScale(optimizer, lr_scheduler, patch_optimizer=True)
```

Comment:
Passing in `lr_scheduler` is a bug?

Reply:
Yea, leftover from a previous commit. I guess we dont ever need the initial scale of Adascale to be set, given that you can pass in arbitrary objects to it.